content/documentation/PinePhone_Pro/Software/Multi-distribution_image.adoc

```diff
@@ -0,0 +1,291 @@
+---
+title: "Multi-distribution image"
+draft: false
+menu:
+  docs:
+    title:
+    parent: "PinePhone_Pro/Software"
+    identifier: "PinePhone_Pro/Software/Multi-distribution_image"
+    weight: 99
+---
+
+:toc:
+
+This article explains how to install a multiple linux distribution enviroment on your PinePhone Pro that is bootable from a microSD card.
+
+== Installing rk2aw
+
+Make sure your phone has the link:/documentation/PinePhone_Pro/Software/Bootloaders/#rk2aw[rk2aw pre-loader] installed to SPI flash. PinePhone Pro models ordered after November 2023 are shipped with rk2aw pre-installed. Otherwise, follow the instructions in this section to install it.
+
+TIP: The multi-boot image _can_ function without rk2aw, but you may need to hold down the **RE** button during boot to disable the SPI flash memory.
+
+Make sure the phone’s SSH server is up and running. On the PinePhone Pro's stock operating system. Then Connect to your phone from your Linux computer using SSH:
+
+----
+ssh USER@PHONEIP
+mkdir -p ~/rk2aw
+cd ~/rk2aw
+curl -O https://xff.cz/kernels/bootloaders-2024.04/ppp.tar.gz
+tar -xvzf ppp.tar.gz -C ~/rk2aw
+cd ~/rk2aw/ppp
+sudo ./spinor-flash-initial-setup.sh
+----
+
+Further instructions can be found on the link:https://xff.cz/kernels/bootloaders-2024.04/ppp/rk2aw/INSTALL[author's website].
+
+== Building
+
+Insert microSD card into the Linux computer. Find out the device name using the command `lsblk` and then make sure there are no signatures or partitions left with the command `sudo wipefs /dev/*[DEVICE]*`, where *[DEVICE]* is the name of the device displayed in the _lsblk_ command. To erase all signatures, type `sudo wipefs --all --force /dev/*[DEVICE]*`.
+
+Overwrite the microSD card with zeroes by either overwriting the first sectors with zeroes (command 1) or the whole device (command 2): 
+
+. `sudo dd if=/dev/zero of=/dev/**[DEVICE]** status=progress bs=32768 count=1`
+. `sudo dd if=/dev/zero of=/dev/**[DEVICE]** status=progress bs=32768 count=$(expr $(lsblk -bno SIZE /dev/[DEVICE] | head -1) \/ 32768)`
+
+=== Partitioning
+
+Set the _PARTSIZE_ variable to the size of your microSD card. A minimum capacity of 64 GB for the microSD card is recommended for 5 distributions.
```

Comment:
Set the _PARTSIZE_ variable to the size of your microSD card. A minimum capacity of 10 GB each distribution is recommended.

Reply:
> That doesn't really make sense. "partition size" as variable and then it uses the full size of the microSD card. And the next sentence is about each partition individually again.

in your revised version:

"Set the PARTSIZE variable to the size of your microSD card. A minimum capacity of 64 GB for the microSD card is recommended for 5 distributions.

For example for a microSD card with 16 GB, set:

PARTSIZE=16G"

How do you fit 16GB size for each distribution in a 16GB card?

i'm confused

Reply:
ok, thanks.